eng/testing/AppleRunnerTemplate.sh

```diff
@@ -11,29 +11,35 @@ XHARNESS_CMD="test"
 XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 XCODE_PATH=$(xcode-select -p)/../..
 
+MACCATALYST="maccatalyst"
+IOS_SIMULATOR="iossimulator"
+IOS="ios"
+TVOS_SIMULATOR="tvossimulator"
+TVOS="tvos"
+
```

Comment:
What do you think about

```suggestion
TARGET_OS=`echo "$TARGET_OS" | tr '[:upper:]' '[:lower:]'`
```

Reply:
Thanks Aleksey, even though `TARGET_OS` value should already be in lower case here, your suggestion to fix it in the template looks more general.